spec/takeaway_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'takeaway'
+
+describe Takeaway do  
+
+	subject(:takeaway) {described_class.new(menu: menu, order: order, sms: sms)} #injecting menu class to allow amy menu methods to be handled in menu class (Order doesn't need to know!) - ## ?? ## Role of value menu?  Doesn't work without, but unsure of role...
+	
+	let(:menu) {double(:menu, print: printed_menu)} #verifying doubles...?!  skipped this step (for now...)
```

Comment:
Metrics/LineLength: Line is too long. [104/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)